sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -70,7 +70,7 @@ public DefaultPodSpec(
             @JsonProperty("share-pid-namespace") Boolean sharePidNamespace,
             @JsonProperty("allow-decommission") Boolean allowDecommission) {
         this(
-                new Builder(Optional.empty(), type, count, tasks) // Assume that Executor URI is already present
```

Comment:
The executor URI comment can be removed since as of this PR that's no longer a thing (hooray!)

Reply:
fixed